app/src/main/java/com/globant/weatherapp/mvp/contracts/WeatherContracts.kt

```diff
@@ -1,10 +1,9 @@
 package com.globant.weatherapp.mvp.contracts
 
-import com.globant.weatherapp.utils.Location
-
 interface WeatherContracts {
     interface Presenter {
         fun initPresenter()
+        fun getCityId(cityName: String): Int?
```

Comment:
it is necessary for this to be nullable?

Reply:
Yes, because the implementation is 
"return locationList.find { it.cityName == cityNameInserted }?.cityId" so I need to use a safe call and the return type has to be nullable.